compiler/onecc-docker/onecc-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need this bash script? Is it for checking if `docker` and `python` exist? We can skip them if you are going to create debian package by having a dependencies. Then, debian tools automatically checks what you want to do.

Please refer to `onecc` codes. (https://github.com/Samsung/ONE/blob/330f9a37cc520c4a5afc91858a8cc2276bb7559a/compiler/one-cmds/onecc#L1)

Reply:
Oh I read that onecc used shell script on top of python code so it can call itself by python
I'll combine shell script and python code later!

I didn't know well about debian package so I didn't consider using dependencies to check python and docker
I'll fix my code according to your advice! Thank you!

Reply:
But we don't use the virtual environment in python script. So we don't need to set the `PY_PATH` I think.

Reply:
I used to thought `onecc-docekr` were placed at `/usr/share/one/`. But it seems not.